Documentation/gettingstarted/istio.rst

```diff
@@ -71,21 +81,20 @@ Cilium's Envoy filters to query the Cilium agent for policy configuration:
 ::
 
     $ awk -f cilium-kube-inject.awk \
-          < istio-cilium-helm/templates/sidecar-injector-configmap.yaml \
-          > istio-cilium-helm/templates/sidecar-injector-configmap.yaml.new && \
-          mv istio-cilium-helm/templates/sidecar-injector-configmap.yaml.new istio-cilium-helm/templates/sidecar-injector-configmap.yaml
+          < ${ISTIO_HOME}/install/kubernetes/helm/istio/templates/sidecar-injector-configmap.yaml \
+          > istio-cilium-helm/templates/sidecar-injector-configmap.yaml
 
 Create an Istio deployment spec, which configures the Cilium-specific variant
 of Pilot, and disables unused services:
 
 ::
 
     $ helm template istio-cilium-helm --name istio --namespace istio-system \
-          --set pilot.image=docker.io/cilium/istio_pilot:1.0.2 \
+          --set pilot.image=docker.io/cilium/istio_pilot:${ISTIO_VERSION} \
           --set sidecarInjectorWebhook.enabled=false \
           --set global.controlPlaneSecurityEnabled=true \
           --set global.mtls.enabled=true \
-          --set global.proxy.image=proxy_debug \
+          --set global.proxy.image=docker.io/cilium/istio_proxy:1.1.3 \
```

Comment:
is this supposed to be `1.1.3` and not `1.1.2`?

Reply:
1.1.3 is correct here.